target/linux/mpc85xx/image/p1020.mk

```diff
@@ -15,6 +15,27 @@ define Device/aerohive_hiveap-330
 endef
 TARGET_DEVICES += aerohive_hiveap-330
 
+define Device/aerohive_hiveap-370
+  DEVICE_VENDOR := Aerohive
+  DEVICE_MODEL := HiveAP-370
+  DEVICE_PACKAGES := kmod-tpm-i2c-atmel
+  BLOCKSIZE := 128k
+  # does not support FIT
+  # KERNEL := kernel-bin | gzip | fit gzip $(KDIR)/image-$$(DEVICE_DTS).dtb
+  # does not support appended dtb with lzma
+  # KERNEL := kernel-bin | append-dtb | lzma
+  # does not support combined dtb (Verifying Checksum ... Bad Data CRC)
+  # KERNEL := kernel-bin | append-dtb | uImage gzip
```

Comment:
You can drop all of these and replace it by a one liner "FIT and LZMA are not supported".

After all, `append-dtb` relies on legacy DTB booting being enabled for the kernel. I'm not sure if PPC even supports this.

Reply:
Probably can remove these comments, as they only serve for informational purposes.